javalib/src/main/scala/java/lang/Runtime.scala

```diff
@@ -67,7 +79,7 @@ class Runtime private () {
       .asInstanceOf[Array[Thread]]
       .sorted(Ordering.by[Thread, Int](-_.getPriority()))
       .foreach { t =>
-        try t.run()
+        try t.start()
```

Comment:
We should revert this change, `runHooksSequential` is run only in the singlethreaded mode

Reply:
Ok, I didn't understand that one because you normally call `start()` on a thread.